Formula/poppler.rb

```diff
@@ -4,15 +4,8 @@ class Poppler < Formula
   head "https://anongit.freedesktop.org/git/poppler/poppler.git"
 
   stable do
-    url "https://poppler.freedesktop.org/poppler-0.68.0.tar.xz"
-    sha256 "f90d04f0fb8df6923ecb0f106ae866cf9f8761bb537ddac64dfb5322763d0e58"
-
-    # https://gitlab.freedesktop.org/poppler/poppler/merge_requests/18
-    # Should be safe to remove on next release but check if merged.
-    patch do
-      url "https://gitlab.freedesktop.org/poppler/poppler/merge_requests/18.diff"
-      sha256 "ef7d7b7fed6799911b7ef925395b8de552aadaf3754d7b02fe3505649518808c"
-    end
+    url "https://poppler.freedesktop.org/poppler-0.69.0.tar.xz"
```

Comment:
The `stable do` block isn't needed, the `url` and `sha256` can be moved above `head`.